crates/burn-sparse/src/decorator/backend.rs

```diff
@@ -0,0 +1,42 @@
+use crate::decorator::FullPrecisionBridge;
+use crate::decorator::SparseRepresentation;
+use burn_tensor::backend::Backend;
+use core::marker::PhantomData;
+use derive_new::new;
+
+/// Tensor backend that extends existing backends with sparse tensor support.
+/// This backend abstracts over all backends, and so lacks the performance of a direct implementation.
+/// Backends implementing SparseDecorator should be used directly where possible.
```

Comment:
Do you mean backends implementing `SparseBackend` here?

Reply:
You are correct, good catch. Most of the decorator code is going to be removed though, given the new API doesn't need it.